Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -244,15 +244,19 @@ private void OnGuiHandler()
             {
                 // Show the active camera and blend
                 var sb = CinemachineDebug.SBFromPool();
+#if CINEMACHINE_UNITY_IMGUI
```

Comment:
This entire method should be removed if !CINEMACHINE_UNITY_IMGUI.  What is the point of the remaining code?